### PR TITLE
Prepare code for rebranding

### PR DESCRIPTION
**Pull request by pruivo**
depends on https://github.com/infinispan/infinispan-quarkus/pull/60

Notes:
* Changed the binary file name from ispn-cli to infinispan-cli. 
* Created zip with name infinispan-cli-<version>.zip

### CODE REVIEW

File: cli/src/main/resources/README.adoc [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+== Installing the CLI as a ${k8.cmd} plugin
+
+You can use the ${infinispan.brand.name} CLI as `${k8.cmd}` client plugin to control the ${infinispan.brand.name} Operator.
+
+Integrating with `${k8.cmd}` lets you install and remove ${infinispan.brand.name} Operator, create and delete ${infinispan.brand.name} clusters, and obtain information about `Infinispan` resources.
+
+To install the CLI as a `${k8.cmd}` plugin, do the following:
+
+. Copy the `${cli.name}`, or create a hard link, to a file named "${k8.cmd}-infinispan".
```
Comment (Crumby) [inline cli/src/main/resources/README.adoc:33]: actually it needs to be `kubectl` if we are not fixing the logic in the CLI itself

File: cli/src/main/resources/README.adoc [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+. Add `${k8.cmd}-infinispan` to your `PATH`.
```
Comment (Crumby) [inline cli/src/main/resources/README.adoc:34]: same here

File: cli/src/main/resources/README.adoc (hunk 1 above)
Comment (pruivo) [reply cli/src/main/resources/README.adoc]: right! both fixed. thanks!

**Comment by pruivo** [comment]
commits squashed :)

**Comment by ryanemerson** [comment]
@pruivo Sorry for the delay. Can you rebase?

**Comment by pruivo** [comment]
rebased.

**Comment by ryanemerson** [comment]
The changes LGTM, just waiting on Jenkins.

**Comment by pruivo** [comment]
updated! fixed the executable name in maven-install plugin

**Comment by pruivo** [comment]
@ryanemerson embedded tests are failing with  `/proto/generated/user.commons.proto`. is it related to my changes?

**Comment by ryanemerson** [comment]
@pruivo I wouldn't have thought so, some changes to Infinispan `master` could have done it. Either way, I'm [rebuilding master](https://ci.infinispan.org/job/Infinispan%20Quarkus/job/master/) to know for sure.

**Comment by ryanemerson** [comment]
Thanks @pruivo and sorry for the delay!

**Pull request merged**